### ISSUE TITLE
rotateZ(0) is mistakenly reported as invalid

### ISSUE THREAD

**Issue by limonte** [csstree/csstree#99]
I'm using `stylelint-csstree-validator` and this line:

```css
transform: rotateZ(0);
 ```

produces an error:

```
Invalid value for `transform`   csstree/validator
```

Changing it to

```css
transform: rotateZ(0deg);
 ```

fixes the issue.

According to https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotateZ `0` is the valid value.

**Comment by lahmatiy** [comment]
In `1.0.0-alpha.30` zero value as valid `<angle>` value was removed, that's [according to spec](https://drafts.csswg.org/css-values-3/#angles):
> Note: For legacy reasons, some uses of `<angle>` allow a bare 0 to mean 0deg. This is not true in general, however, and will not occur in future uses of the `<angle>` type.

Transform function syntaxes were updated to explicit allow zero usage, and it's already [merged to mdn/data](https://github.com/mdn/data/pull/376). However mdn/data with the fix doesn't published at the moment.

I'm going to make some fixes in lexer, and relax `<angle>` check (by allowing zero) if `mdn/data` folks will not publish a new version.

**Comment by lahmatiy** [comment]
`1.0.0-alpha.31` with updated `mdn/data` is out, see [release notes](https://github.com/csstree/csstree/releases/tag/v1.0.0-alpha.31) for details
New version of `stylelint-csstree-validator` has been published as well – [1.5.0](https://github.com/csstree/stylelint-validator/releases/tag/v1.5.0)

**Comment by limonte** [comment]
Thank you @lahmatiy!
